stl/src/special_math.cpp

```diff
@@ -14,6 +14,7 @@
 #pragma warning(disable : 4702) // unreachable code
 
 #define BOOST_CONFIG_SUPPRESS_OUTDATED_MESSAGE
+#define BOOST_CHRONO_HEADER_ONLY

```

Comment:
Could we move this up a line for lexicographically-sorted goodness?

Reply:
Done